src/Bootstrappers/RegisterExceptionHandler.php

```diff
@@ -94,7 +94,13 @@ public function handleError($level, $message, $file = '', $line = 0, $context =
     {
         $exception = new ErrorException($message, 0, $level, $file, $line);
 
-        if ($level === E_USER_NOTICE || $level === E_USER_DEPRECATED) {
+        $errorsToReportOnly = $this->app->get('config')->get('app.errors.reportOnly') ?: [
+            E_USER_NOTICE,
+            E_USER_DEPRECATED,
+            E_DEPRECATED
+        ];
```

Comment:
Thoughts on just using the default parameter on the `get()` method rather than `?:`?

```suggestion
        $errorsToReportOnly = $this->app->get('config')->get('app.errors.reportOnly', [
            E_USER_NOTICE,
            E_USER_DEPRECATED,
            E_DEPRECATED
        ]);
```